Automattic/jetpack
# Build process: new npm module that allows gulp-uglify to process es6

#### Changes proposed in this Pull Request:


Adds a new npm module [uglify-es](https://www.npmjs.com/package/uglify-es) that allows `gulp-uglify` to process es6. 

It was broken when we [updated](https://github.com/Automattic/jetpack/pull/10849) to gulp-uglify 3.0, as it now depends on the default uglifyJS within it.

This is now the [suggested method](https://github.com/terinjokes/gulp-uglify#using-a-different-uglifyjs) for es6 via gulp-uglify.

#### Testing instructions:





- `yarn install`
- run `yarn build-production-client`
- Verify no errors
- Check `/_inc/build/admin.js` file. It should be minified.

run the dev commands as well: 
- `yarn build`
- `yarn watch`
- verify that nothing is broken in the build, and the admin page renders fine. 

#### Proposed changelog entry for your changes:


none
